tests/unit/amo/pages/TestAddonInfo.js

```diff
@@ -344,10 +344,16 @@ describe(__filename, () => {
     const dispatch = jest.spyOn(store, 'dispatch');
     render();
 
-    expect(dispatch).toHaveBeenCalledWith(
+    // Expect only the LOCATION_CHANGE action which happens twice.
```

Comment:
Here and in `TestBlock.js` and `TestHome.js` below, add a quick sentence explaining that's it because of `render()` as you did in `TestSearchTools.js` because that's not straightforward if you're not familiar with all this...

Reply:
Done!